tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -30,6 +30,30 @@ load(
 )
 load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
 
+# The Xcode version from which that has support for deterministic mode
+_SUPPORTS_DETERMINISTIC_MODE = "10.2"
+
+def _compare_versions(dv1, v2):
+    """Returns value is <0, 0, >0 depending on DottedVersion dv1 comparison to string v2."""
```

Comment:
nit: "Return value is..." or "Returns ..."

Reply:
Fixed. Thanks.